tests/test_login.py

```diff
@@ -12,8 +13,9 @@ class TestLogin(object):
 
     _fxa_logged_in_indicator_locator = (By.ID, 'loggedin')
 
-    def test_user_can_sign_in(self, base_url, selenium, dev_account, click_login):
-        fxa = WebDriverFxA(base_url, selenium)
+    def test_user_can_sign_in(self, base_url, selenium, dev_account, click_login, timeout):
+        fxa = WebDriverFxA(base_url, selenium, timeout)
         fxa.sign_in(dev_account.email, dev_account.password)
-        WebDriverWait(selenium, 20).until(
-            lambda s: s.find_element(*self._fxa_logged_in_indicator_locator).is_displayed())
+        # We sometimes need to wait longer than the standard 10 seconds
+        Wait(selenium, 20).until(
```

Comment:
I would just set the default to 20 seconds. It saves having two values, and it's not really going to cost us much.